samples/FakeItEasy.Examples.ruleset

```diff
@@ -13,5 +13,6 @@
     <Rule Id="CA1824" Action="None" />
     <Rule Id="CA2201" Action="None" />
     <Rule Id="CA2210" Action="None" />
+    <Rule Id="SA0001" Action="None" />

```

Comment:
1.1.0-beta006 doesn't like it when a project is set to not generate XML docs. We don't really need them for the Examples or Tests.